src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -165,6 +173,15 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
     [joinMessage, onAcceptJoinMessage, showJoinMessage, showLockedMessage]
   );
 
+  const emptyTables = tables.filter(
+    (table) =>
+      !recentVenueUsers.filter(
+        (u) => u.data?.[venueName]?.table === table.reference
+      ).length
+  );
```

Comment:
`useMemo` ?

Reply:
fixed in d925f6fc105f25f1e64646294106b771d64b8bba